rating-service/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,3 @@
+spring.cloud.config.discovery.enabled=true
```

Comment:
Unfortunately not, bootstrap file is loaded as first one, if we put properties to application.
FIrstly will be loaded anyway default link with port 8888, checked such effect when set configuration server to some custom port as 9999 i.e.. In such case works only if properties of configuration server are read from bootstrap property file.